pyo3-derive-backend/src/pyclass.rs

```diff
@@ -213,7 +213,7 @@ fn parse_descriptors(item: &mut syn::Field) -> syn::Result<Vec<FnType>> {
 fn impl_inventory(cls: &syn::Ident) -> TokenStream {
     // Try to build a unique type that gives a hint about it's function when
     // it comes up in error messages
-    let name = cls.to_string() + "GeneratedPyo3Inventory";
+    let name = format!("Pyo3MethodsInventoryFor{}", cls);
```

Comment:
```suggestion
    let name = format!("PyMethodsOf{}", cls);
```

Longer name is also fine, just wondered if I could find a shorter one which worked.

Reply:
Since this is a *generated* class, I think it's better if a user can understand that *this is generated* from the name if he or she uses `cargo expand`.

Reply:
That's fair. `PyO3GeneratedMethodsFor{}` ? Your choice of name is also fine.